src/org/parosproxy/paros/extension/report/ReportGenerator.java

```diff
@@ -126,7 +127,7 @@ public static File stringToHtml(String inxml, String infilexsl, String outfilena
 					bw = new BufferedWriter(new FileWriter(outfilename + "-orig.xml"));
 					bw.write(inxml);
 				} catch (IOException e2) {
-					logger.error(e.getMessage(), e);
+					logger.error(Constant.messages.getString("report.write.diskspace.error.log.message"), e);

```

Comment:
I think it should be changed to the exception message, but looking again it should be another message, something like "Failed to write debug XML file:", which is more informative.
Also, the view warn should be in the outer catch block, otherwise there's no warn if the "debug" file is correctly saved.